### PR TITLE
Update README for Moby Project

### PR DESCRIPTION
**Pull request by justincormack**
As described in #35115 Completing the Moby Transition, we would like to complete the transition
stage that we began a while back. This tries to clarify what and who this project is for.

Signed-off-by: Justin Cormack <justin.cormack@docker.com>

whale

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,38 @@
...
 
 ![Moby Project logo](docs/static_files/moby-project-logo.png "The Moby Project")
 
-Moby is an open-source project created by Docker to advance the software containerization movement.
-It provides a “Lego set” of dozens of components, the framework for assembling them into custom container-based systems, and a place for all container enthusiasts to experiment and exchange ideas.
-
-# Moby
-
-## Overview
+Moby is an open-source project created by Docker to enable and accelerate software containerization.
```
Comment (AkihiroSuda) [inline README.md]: Is "Moby" (as a software name) also the successor of (the daemon side of) Docker Engine?

File: README.md [hunk 2]
```diff
@@ -1,70 +1,38 @@
-### Docker users, see [Moby and Docker](https://mobyproject.org/#moby-and-docker) to clarify the relationship between the projects
```
Comment (AkihiroSuda) [inline README.md]: I think this sentence is still good to keep

File: README.md (hunk 1 above)
Comment (justincormack) [reply README.md]: Moby is the name of the whole project, which includes the moby github org, and other projects (linuxkit etc) that have their own orgs - everything mentioned on mobyproject.org. But I am also using it as a name for this repo, the Moby Project central repo, which is an integration point for a lot of pieces, including the current daemon. The actual binary naming is discussed in #34226

File: README.md [hunk 3]
```diff
@@ -1,70 +1,38 @@
...
-
-### Docker maintainers and contributors, see [Transitioning to Moby](#transitioning-to-moby) for more details
-
 The Moby Project
 ================
 
 ![Moby Project logo](docs/static_files/moby-project-logo.png "The Moby Project")
 
-Moby is an open-source project created by Docker to advance the software containerization movement.
-It provides a “Lego set” of dozens of components, the framework for assembling them into custom container-based systems, and a place for all container enthusiasts to experiment and exchange ideas.
```
Comment (AkihiroSuda) [inline README.md]: I liked the concept of "lego set" 😢

Comment (justincormack) [reply README.md]: @thaJeztah wasnt sure if we should be using trademarks like that... but it still is!

Comment (thaJeztah) [reply README.md]: Yes, I do like it, but wasn't sure if we were allowed to use it (without mentioning trademarks in a footnote 😅)

File: README.md (hunk 1 above)
Comment (tiborvass) [reply README.md]: @AkihiroSuda But to clearly answer you: no, moby ≠ engine, but moby > engine.

Comment (AkihiroSuda) [reply README.md]: @justincormack @tiborvass ok, thank you for explanation

**Comment by yongtang** [comment]
Looks good to me though your picture makes me dizzy 😅

File: README.md (hunk 2 above)
Comment (justincormack) [reply README.md]: It is still correct, but I really wanted to not begin with something about Docker, I wanted the project to stand on its own. I think for anyone reading this it should be clear enough.

**Review by AkihiroSuda — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (justincormack) [reply README.md]: Apparently there will be Lego at DockerCon EU though!

**Review by thaJeztah — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (shykes) [reply README.md]: I think this sentence falls pretty clearly under "fair use". I propose we re-open a PR to bring it back.

Comment (yongtang) [reply README.md]: 👍  for bring back "Lego set". Added a PR #35156.

Comment (justincormack) [reply README.md]: merged that, thanks

**Pull request merged**